src/bootstrap/defaults/config.library.toml

```diff
@@ -1,13 +1,10 @@
 # These defaults are meant for contributors to the standard library and documentation.
-[build]
-# When building the standard library, you almost never want to build the compiler itself.
-build-stage = 0
-test-stage = 0
-bench-stage = 0
```

Comment:
This doesn't make sense anymore, right? @RalfJung

Reply:
You mean because it'll download rustc so testing with stage 1 is fine now? It's still some extra work though since it builds std twice. So I'd say we should only change this once the big std bootstrap change is done.

But no strong opinion, I don't have good insight into how people use these profiles.

Reply:
Ah right. In that case, we should probably at least preserve these for now.